src/karmen_backend/server/routes/printjobs.py

```diff
@@ -177,13 +172,13 @@ def printjobs_list(org_uuid):
     return jsonify(response), 200
 
 
-@app.route("/organizations/<org_uuid>/printjobs/<uuid>", methods=["GET"])
+# @app.route("/organizations/<org_uuid>/printjobs/<printjob_uuid>", methods=["GET"])
 @jwt_force_password_change
 @validate_org_access()
 @cross_origin()
-def printjob_detail(org_uuid, uuid):
-    validate_uuid(uuid)
-    printjob = printjobs.get_printjob(uuid)
+def printjob_detail(org_uuid, printjob_uuid):
+    validate_uuid(printjob_uuid)
```

Comment:
Is this validation still necessary?

Reply:
We had some issues few version back, where there were some invalid UUIDs generated, and the pythons uuid module, instead of throwing an error, changed them to something valid. without throwing any error. This then caused some null selects from db. They should have been fixed during migrations, I don't know how Jirka decided to solve that and why is this here. If we want to clean that up, we should make it another task. This is happening in more places.

Reply:
I added new card regarding this issue to inbox